SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestMultipleOwnership/SpatialTestMultipleOwnership.cpp

```diff
@@ -212,4 +223,13 @@ void ASpatialTestMultipleOwnership::PrepareTest()
 			FinishStep();
 		},
 		10.0f);
+
+	// The server makes the player controllers possess the original pawns they had before the start of the test.
+	AddStep(TEXT("SpatialTestMultipleOwnershipCleanup"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		for (const auto& Pair : OriginalPossessedPawns)
```

Comment:
Are we reasonably sure Server1 will have authority over all pawns and controllers? Maybe, assert here?

Reply:
yes, this is single-server only

Reply:
I think there's actually already an error logged from spatial if you try to possess across servers